### ISSUE TITLE
Debug GH actions caching on Windows.

### ISSUE THREAD

**Issue by jpfeuffer** [OpenMS/OpenMS#6624]
Something has to be wrong.

**Comment by timosachsenberg** [comment]
please add details if the issue is not resolved

**Comment by jpfeuffer** [comment]
Barely any time savings.

**Comment by timosachsenberg** [comment]
still a thing?

**Comment by jpfeuffer** [comment]
Yes, still by far the longest build. Not sure if it caches correctly. But also not sure what to expect from MSVC.
I dont remember how long the builds were before caching anymore.
